R/container.R

```diff
@@ -0,0 +1,174 @@
+#' Declare post-processing for model predictions
+#'
+#' @param mode The model's mode, one of `"unknown"`, `"classification"`, or
+#' `"regression"`. Modes of `"censored regression"` are not currently supported.
+#' @param type The model sub-type. Possible values are `"unknown"`, `"regression"`,
+#' `"binary"`, or `"multiclass"`.
+#' @param outcome The name of the outcome variable.
```

Comment:
Did you mean {tidyselect} instead of {tidypredict}? 

We do use tidyselect in `fit.container()` and the character defaults are there for that. I can make the man page reflect that it will be a tidyselect selector.

Reply:
yes, tidyselect